Octokit/Clients/RepositoriesClient.cs

```diff
@@ -246,6 +261,7 @@ public Task<Repository> Edit(long repositoryId, RepositoryUpdate update)
         /// <param name="name">The name of the repository</param>
         /// <exception cref="ApiException">Thrown when a general API error occurs.</exception>
         /// <returns>A <see cref="Repository"/></returns>
+        [Preview(AcceptHeaders.TemplateRepositoryPreviewCodeName)]
```

Comment:
In addition to this link, you need to pass in the accepts header to ensure the `Repository` response returns the additional field:

```c#
[Preview(AcceptHeaders.TemplateRepositoryPreviewCodeName)]
[ManualRoute("GET", "/repositories/{id}")]
public Task<Repository> Get(long repositoryId)
{
     return ApiConnection.Get<Repository>(ApiUrls.Repository(repositoryId), null, AcceptHeaders.TemplateRepositoryPreview);
}
```

This will also require the corresponding tests to be updated.